packages/connect-query/src/create-unary-functions.ts

```diff
@@ -76,12 +75,12 @@ export interface UnaryHooks<I extends Message<I>, O extends Message<O>> {
    * The caveat being that if you go the route of using `createUseQueryOptions` you must provide transport.  You can get transport from the `useTransport` export.  If you cannot use hooks to retrieve transport, then look at the documentation for `TransportProvider` to learn more about how to use Connect-Web's createConnectTransport` or `createGrpcWebTransport`APIs.
    */
```

Comment:
Should this comment be updated based on the changes below? Seems like the transport is now optional but this comment says it must be provided.

Reply:
Ah good catch. No, it's not technically required (if the service was created with a provided transport up front). But in most cases, it must be provided. I'll update the comment.

Reply:
I've updated this a little, though honestly I feel the fact this isn't just a mandatory field a little annoying. It's only this way since `createUnaryFunctions` itself takes an optional transport which this can fallback to.